src/modules/QtQuick.Controls/ComboBox.js

```diff
@@ -43,29 +49,72 @@ QmlWeb.registerQmlType({
     return this.model[index];
   }
   $updateImpl() {
-    this.currentIndex = 0;
     this.count = this.model.length;
-    const entries = [];
-    for (let i = 0; i < this.count; i++) {
-      const elt = this.model[i];
-      //if (elt instanceof Array) { // TODO - optgroups? update model !
-      //    var count_i = elt.length;
-      //    for (var j = 0; j < count_i; j++)
-      //        html += "<option>" + elt[j] + "</option>";
-      //}
-      //else
-      entries.push(`<option>${elt}</option>`);
-    }
-    // TODO: remove innerHTML, port to DOM
-    this.dom.innerHTML = `<select>${entries.join("")}</select>`;
+
+    // TODO change innerHTML to DOM
+    this.dom.innerHTML = "<select></select>";
```

Comment:
This is nice. Now, constructing DOM via setting `innerHTML` is not needed here — I will remove that after merging.